builtins/amp-img.js

```diff
@@ -85,7 +87,11 @@ export function installImg(win) {
       }
       this.img_.setAttribute('src', src);
 
-      return loadPromise(this.img_);
+      return loadPromise(this.img_).catch(error => {
+        if (isProxyOrigin(src)) {
```

Comment:
I don't think this will work, because these urls are relative. You want to check the document URL.

Please add a comment to the whole thing why we are doing it.